docs/README.md

```diff
@@ -0,0 +1,13 @@
+# Documentation
+
+- [How to add a new rule](./how-to-add-a-new-rule.md)
+- [How to write tests for plugins](./how-to-write-tests-for-plugins.md)
+- [Compatibility](./compatibility.md)
+- [How it works](./how-it-works.md)
+- [How to debug](./how-to-debug.md)
```

Comment:
Is the document necessary?

Reply:
Maybe they are not necessarily listed in README.